tests/PKSim.Tests/Core/PopulationAnalysisFieldMapperSpecs.cs

```diff
@@ -88,7 +90,7 @@ protected override Task Context()
          A.CallTo(() => _groupingDefinitionMapper.MapToSnapshot(_groupingDefinition)).Returns(_snapshotGroupingDefinition);
 
          A.CallTo(() => _dimensionFactory.Dimension(_dimension.Name)).Returns(_dimension);
-         A.CallTo(() => _dimensionFactory.MergedDimensionFor(A<DataColumn>._)).Returns(_dimension);
```

Comment:
and before it was returning the same instance. now the test returns something else